web/libs/editor/src/mixins/DynamicChildrenMixin.js

```diff
@@ -51,9 +62,13 @@ const DynamicChildrenMixin = types
         // we may need to rewrite this, initRoot and the other related methods
         // (actually a lot of them) to work asynchronously as well
 
-        setTimeout(() => {
+        if (ff.isActive(FF_DEV_3391)) {
```

Comment:
why do you need this change? I'm not sure about side effects of it being both sync and async.

Reply:
The setTimeout is forcing this code to be executed on the next tick so, by the time it reaches the point to populate the array, the store is always initialized.

If we are planning to get rid of the flag in the short term we should be able to remove the setTimeout and the the ff check on store initialization, it works

Reply:
Why did we add it in the first place by the way?

Reply:
Selenium passed, resolving